frontend/src/components/panels/NodeGraph.vue

```diff
@@ -632,6 +647,38 @@ export default defineComponent({
 					}
 				}
 				this.editor.instance.moveSelectedNodes(this.draggingNodes.roundX, this.draggingNodes.roundY);
+
+				if (this.selected.length === 1) {
+					const id = this.selected[0];
+					const selectedNode = containerBounds.querySelector(` [data-node="${String(id)}"]`);
```

Comment:
Nit: extraneous space at the start of the template string.

While you're here, would you mind giving the code a little breathing room by spacing out the related blocks of lines?

Also while you're working in this event handler, would you mind adding something for closing the Add Nodes menu if you click out of it while it's open? Thanks :)